### ISSUE TITLE
Bad example in file doc/Language/variables.pod

### ISSUE THREAD

**Issue by tbrowder** [perl6/doc#527]
Around line 946 this example doesn't work:

    use XML;
    XML.parse(slurp "filename.xml")
    say $/;

I'm trying to get an equivalent example but haven't succeeded so far--inputs welcome!

**Comment by DabeDotCom** [comment]
I presume that example is bad [apart from the missing semicolon] because it depends on non-core modules... yet there *AREN'T* really any "core" grammars — apart from Perl6 itself!

Otherwise, just:

```
use XML::Grammar;    # From "panda install XML"
XML::Grammar.parse(slurp "filename.xml");
say $/;
```

?

**Comment by tbrowder** [comment]
On Thursday, May 19, 2016, DabeDotCom <notifications@github.com> wrote:

> I presume that example is bad [apart from the missing semicolon] because
> it depends on non-core modules... yet there *AREN'T* really any "core"
> grammars — apart from Perl6 itself!
>
> Otherwise, just:
>
> use XML::Grammar;    # From "panda install XML"
> XML::Grammar.parse(slurp "filename.xml");
> say $/;
>
> ?
>

Yeah, I had tried the XML module but didn't see the Grammar module.
Anyway, that doesn't look much good either.

I started defining a simple grammar to make a parser example but don't yet
know enough to make it work.

Thanks for the interest, though.

Best regards,

-Tom

>

**Comment by tbrowder** [comment]
closed with merged pull #540
